### PR TITLE
Remove posthog cloud

### PR DESCRIPTION
**Pull request by MananTank**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
The focus of this PR is updating dependencies and removing PostHog Cloud integration.

### Detailed summary
- Updated `posthog-js` to `posthog-js-opensource`
- Removed PostHog Cloud integration in `_app.tsx`
- Adjusted script source in `next.config.js`
- Updated various package versions in `pnpm-lock.yaml`

> The following files were skipped due to too many changes: `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by MananTank** [comment]
* **#4671** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4671?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MananTank and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/pages/_document.tsx [hunk 1]
```diff
@@ -10,8 +10,6 @@ class ConsoleDocument extends Document {
           {/* preconnect to domains we know we'll be using */}
           <link rel="preconnect" href="https://a.thirdweb.com" />
           <link rel="dns-prefetch" href="https://a.thirdweb.com" />
-          <link rel="preconnect" href="https://pg.paper.xyz" />
```
Comment (jnsdls) [inline apps/dashboard/src/pages/_document.tsx:13]: also remove it from `CSP` in next.config.js

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/pages/_document.tsx (hunk 1 above)
Comment (MananTank) [reply apps/dashboard/src/pages/_document.tsx:13]: removed, good catch

**Review by jnsdls — APPROVED** [review]

**Comment by MananTank** [comment]
* **#4671** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4671?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @MananTank and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Pull request merged**